bin/compile

```diff
@@ -85,10 +85,17 @@ urlFor() {
     esac
 }
 
-# Expand to supported versions of Go, see data.json for supported expansions.
-# All others are returned as is.
+# Expand a version identifier to supported versions of Go. If the version
+# identifier starts with one or more digits, then "go" is prepended before
+# expansion. So `1.9` => `go1.9`, `1.9.4` => `go1.9.4`, `devel` => `devel`. See
+# data.json for supported expansions. All others are returned as is without go
+# prepended to it.
 expandVer() {
-    echo $(<${DataJSON} jq -r 'if .Go.VersionExpansion."'${1}'" then .Go.VersionExpansion."'${1}'" else "'${1}'" end')
+    local v="${1}"
+    if [[ ${v} =~ ^[[:digit:]]+ ]]; then
```

Comment:
Should things be quoted here?

Reply:
The variable, sure, but not the regex: 1e01816509e5365b058f2a39c39da6b6461f62cf